test/fixers/test_buildifier_fixer_callback.vader

```diff
@@ -0,0 +1,24 @@
+Before:
+  Save g:ale_bzl_buildifier_executable
+
+  " Use an invalid global executable, so we don't match it.
+  let g:ale_bzl_buildifier_executable = 'xxxinvalid'
+
+  call ale#test#SetDirectory('/testplugin/test/fixers')
+  silent cd ..
+  silent cd command_callback
+  let g:dir = getcwd()
+
+After:
+  Restore
+
+  call ale#test#RestoreDirectory()
+
+Execute(The buildifier callback should return the correct default values):
+  call ale#test#SetFilename('bzl_paths/BUILD')
```

Comment:
Consider checking that we work out of the box with:
* `bzl_paths/BUILD.bazel`
* `bzl_paths/rules.bzl`

which are also a common extensions, which could work with this.

If you decide to add support for starlark scripts then consider doing the same for `.star` files.

Reply:
I modified my unit tests so that multiple different file types associated with Bazel are tested. 

At least, _I think_ that's what I'm testing. My knowledge of vimscript is next to non-existent, and I don't know the ale API at all, so they may not be correct :grin: